Trinity-Plumbing-App/src/app/components/store-page/store-page.component.html

```diff
@@ -0,0 +1,174 @@
+<div class="con">
+  <div class="head">
+    <h1>Store</h1>
+    <p>Home / Tools</p>
+  </div>
+
+
+<div class="store">
+  <div class="categories">
+    <div class="top">
+      <h2>Categories Menu</h2>
+      <hr>
+    </div>
+    <div class="mid">
+      <p>Construction</p>
+      <p>Hardware</p>
+      <p>Plumber Tools</p>
+      <p>Tools</p>
+    </div>
+
+    <div class="top">
+      <h2>Availability</h2>
+      <hr>
+    </div>
+    <div class="mid">
+      <p>In Stock (2)</p>
+      <p>Out Of Stock</p>
+      <div class="btns">
+        <button>Clear</button>
+        <button>Apply</button>
+      </div>
+    </div>
+
+    <div class="top">
+      <h2>Best Sellers</h2>
+      <hr>
+    </div>
+
+    <div class="bs_Box">
+      <div class="imgBx">
+        <img src="../../../assets/global/header_img.jpg" alt="">
+      </div>
+      <div class="icons">
+
+      </div>
+      <div class="dets">
+        <p class="name">Kitchen Pipe Chrome</p>
+        <p class="price">$8,500.00</p>
+      </div>
+    </div>
+
+    <div class="arrs">
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Pro 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. --><path d="M9.4 233.4c-12.5 12.5-12.5 32.8 0 45.3l160 160c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L109.2 288 416 288c17.7 0 32-14.3 32-32s-14.3-32-32-32l-306.7 0L214.6 118.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0l-160 160z"/></svg>
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Pro 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. --><path d="M438.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-160-160c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L338.8 224 32 224c-17.7 0-32 14.3-32 32s14.3 32 32 32l306.7 0L233.4 393.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0l160-160z"/></svg>
+    </div>
+
+  </div>
+
+  <div class="items">
+
+    <div class="search">
+      <label for="">Sort By </label>
+      <select name="" id="">
+        <option value="">Featured</option>
+      </select>
+      <!--
+        <div class="icon">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Pro 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. <path d="M416 208c0 45.9-14.9 88.3-40 122.7L502.6 457.4c12.5 12.5 12.5 32.8 0 45.3s-32.8 12.5-45.3 0L330.7 376c-34.4 25.2-76.8 40-122.7 40C93.1 416 0 322.9 0 208S93.1 0 208 0S416 93.1 416 208zM208 352c79.5 0 144-64.5 144-144s-64.5-144-144-144S64 128.5 64 208s64.5 144 144 144z"/></svg>
+      </div>
+    -->
+    </div>
+
+    <div class="itemCon">
+
+      <div class="box">
```

Comment:
break this up into a smaller items.
when we integrate the data from the API we just want to pass data in and populate them.
create a smaller component and loop through an array to render. we will replace array with data from api later